components/portal/misc/filters.js

```diff
@@ -85,9 +88,18 @@ define(['angular'], function(angular) {
     })
     .filter('urlToTarget', function() {
       return function(url) {
-        var result = '_self';
+        var result = TARGET_SELF;
         if (url && url.indexOf('//') > -1) {
-          result = '_blank';
+          result = TARGET_BLANK;
```

Comment:
:art: could this be directly `return`ed, rather than stored to intermediate `result`?